### PR TITLE
feat: add tuya-panel-mall-toolbar-indicator

### PR DESCRIPTION
**Pull request by Nfs-oh**
## Description

add the feature tuya-panel-mall-toobar-indicator.

## Type of change

Please select the relevant option.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [✅] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: packages/tuya-panel-topbar-mall-indicator/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  titlePress?: () => void;
+  containerStyle?: any;
+  imageStyle?: any;
+  rightPress?: () => void;
+  rightActionType?: string;
+};
+
+const ToolBarMallIndicator: React.FC<IProps> = ({
+  title,
+  titlePress,
```
Comment (ShinyLeee) [inline packages/tuya-panel-topbar-mall-indicator/src/index.tsx:20]: onTitlePress
onRightPress

语义更好一些

**Review by ShinyLeee — CHANGES_REQUESTED** [review]
补下单元测试

**Comment by ZengBeauty** [comment]
yarn build 失败

File: packages/tuya-panel-topbar-mall-indicator/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+  return (
+    <TopBar.Container>
+      <TopBar.Action name="backIos" onPress={handleBack} />
+      <TopBar.Content
+        title={title}
+        subTitle={subTitle}
+        onPress={() => {
+          onTitlePress && onTitlePress();
+        }}
```
Comment (ShinyLeee) [inline packages/tuya-panel-topbar-mall-indicator/src/index.tsx:110]: 借鉴下 TopBar 的实现，把剩余的 TopBar.Content props 传递进去，可拓展性高一些

**Comment by Nfs-oh** [comment]
the branch is so complement, so delete it add create a new branch

**Pull request closed without merging**